### PR TITLE
App views

### PR DESCRIPTION
**Pull request by lengushuru**
### Hello CR 👋 
**In this PR the following changes are suggested for merge 👇 :**

-  Implemented the design from the sneak peek wireframes.
- Add posts views
-  Add styling to the views.
- Add users views
-  Fix Linter errors.
-  Follow GitFlow.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,19 @@
...
+    <h2><%= user.name %></h2>
+  </div>
+  <p class="post-count">Number of Posts:<%= user.posts_counter%></p>
+  </a>
+</li>
+<% end %>
+
+</ul>
+
+</section>
```
Comment (zieeco) [inline app/views/users/index.html.erb:19]: - Hey, after creating a user, the user doesn't seem to be created and it's empty in the console. please, kindly check this to fix this issue.
- And this has affected every other view, e.g, show, post index, and post-show page. kindly fix it to ensure each page shows as expected.

**NOTE** _Please, kindly fixes all the pages and ensure they work as expected!_

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @lengushuru,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice adding the app view.✔️
- Good commit message. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (lengushuru) [reply app/views/users/index.html.erb:19]: thank you @zieeco for suggestion I will work on fixing the bug.

**Review by brytebee — APPROVED** [review]
Hi, @lengushuru 👋,

##

### Highlights💡
- Good implementation of the previous reviewer's suggestions.✔️ 
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**